src/features/embalm/stepContent/hooks/useEnterToNextStep.ts

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from 'react';
+import { Step } from 'store/embalm/reducer';
+import { useSarcophagusParameters } from './useSarcophagusParameters';
+import { useStepContent } from './useStepContent';
+
+/**
+ * Adds an "Enter" keydown event listener that goes to the 
+ * next step in the create sarco form if `canGoNext` is true
+ * */
+export function useEnterToNextStep() {
+    const { goToNext, currentStep } = useStepContent();
+    const { sarcophagusParameters } = useSarcophagusParameters();
+
+    useEffect(() => {
+        if (currentStep === Step.CreateSarcophagus) return;
+
+        const currentStepParams = sarcophagusParameters.find(s => s.step === currentStep);
+
+        const canGoNext = currentStep === Step.NameSarcophagus ?
+            !currentStepParams?.error && !sarcophagusParameters.find(s => s.name === 'RESURRECTION')?.error :
+            !currentStepParams?.error;
```

Comment:
See PR description for reasoning behind this logic.

It does feel a bit hacky though - problem being that two "steps" are actually part of one step in the UI/UX.